source/customizing_look_and_feel/04_configuring_data_visualizations/basemaps.rst

```diff
@@ -1,39 +1,166 @@
-Choosing a base map
-===================
+Adding basemaps
+===============
 
-Mapbox integration
-------------------
-
-`Mapbox <https://www.mapbox.com/>`_ is a mapping platform that allows you to build and design your own maps; they also
-provide a dozen different map styles that you can directly use. If you have a Mapbox account, you can configure it into
-OpenDataSoft so that your users can display your data over these map styles.
+Basemaps are the canvas on which geographic data are represented. They are used for any map displayed on OpenDataSoft, whether it is that of the visualization map of a dataset or of the Map Builder.
```

Comment:
"whether it is that of the visualization map of a dataset or of the Map Builder" => "like the visualization map of a dataset or the Map Builder"